yelp_kafka/config.py

```diff
@@ -37,7 +37,10 @@
 from yelp_kafka.utils import memoized
 
 
-DEFAULT_KAFKA_TOPOLOGY_BASE_PATH = '/nail/etc/kafka_discovery'
+DEFAULT_KAFKA_TOPOLOGY_BASE_PATH = '/nail/srv/configs/kafka_discovery_configs'
+# Use /etc/kafka_discovery if /nail/srv/configs/kafka_discovery does not exist
+if not os.path.isdir(DEFAULT_KAFKA_TOPOLOGY_BASE_PATH):
+    DEFAULT_KAFKA_TOPOLOGY_BASE_PATH = '/etc/kafka_discovery'
```

Comment:
Do you think we we should fall back to using the `/nail/etc/kafka_discovery` (status quo Kafka topology base path) directory rather than the `/etc/kafka_discovery` directory? I suppose it shouldn't matter since `/etc/kafka_discovery` symlinks to  `/nail/etc/kafka_discovery` in most places, but I'm not sure if we currently guarantee that Puppet will create this symlink on _all_ hosts which rely on `yelp_kafka`.

```
└─▶ ls -al /etc/kafka_discovery
lrwxrwxrwx 1 root root 25 Aug 20 05:02 /etc/kafka_discovery -> /nail/etc/kafka_discovery
```

Reply:
Good catch!